manifests/t/TopazLabsLLC/TopazGigapixelAI/6.1.0/TopazLabsLLC.TopazGigapixelAI.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: TopazLabsLLC.TopazGigapixelAI

```

```suggestion
PackageIdentifier: TopazLabs.TopazGigapixelAI
```
Edit filename and foldername as well